pkg/metrics/scaledjob_prometheus_metrics.go

```diff
@@ -0,0 +1,124 @@
+package metrics
```

Comment:
Adding Prometheus metrics for ScaledJob is great thing. But could you please do that in a separate PR? As I understand it, it is not required for the current feature?

And is there a particular reason why are you creating a separate Prometheus metrics server for ScaledJob? Extending the current one with new metrics is not possible?

Reply:
Good catch... I didn't notice it. I'll remove.